src/prompt/tests.py

```diff
@@ -1 +1,164 @@
-# Create your tests here.
+from django.test import Client
+from django.urls import reverse
+from rest_framework.test import APITestCase
+import json
+from prompt.models import Prompt
+from api.tests import PearTestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+
+
+class PromptTestCase(APITestCase):
+
+    def setUp(self):
+        pear_test_case = PearTestCase()
+        pear_test_case.setUp()
+        self.user_token = pear_test_case._create_user_and_login()
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.user_token)
+
+    def populate(self):
+        """populates the database"""
+        url = reverse("populate")
+        return self.client.post(url, {"filenames": ["pear_prompts.txt"]}, format='json')
+
+    def test_populate_prompts(self):
+        """tests that the code populates the 27 prompts in pear_prompts.txt"""
+
+        response = self.populate()
+        content = json.loads(response.content)
+        self.assertTrue(content.get("success"))
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual(Prompt.objects.count(), 27)
+
+    def test_repopulate(self):
+        """test repopulate"""
+
+    def test_get_all_prompts(self):
+        """tests getting all the promts"""
+        populate = self.populate()
+
+        url = reverse("prompts")
+        response = self.client.get(url, format='json')
+        content = json.loads(response.content)
+        self.assertTrue(content.get("success"))
+        self.assertEqual(response.status_code, 200)
+
+    def test_get_prompt_by_id(self):
+        """tests getting a promt by a valid id"""
+        poulate = self.populate()
+
+        url = reverse("prompt", args=[1])
+        response = self.client.get(url, format='json')
+        content = json.loads(response.content)
+        self.assertTrue(content.get("success"))
```

Comment:
I see this logic quite a lot; have you considered making a helper function that asserts the contents of content?